### PR TITLE
Added About Kathmandu City

### PR DESCRIPTION
**Pull request by rabingaire**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
-        Nepal is a himalayan country situated between India and China.
-    </body>
-
-</html>
\ No newline at end of file
+  <head>
+    <title>Nepal</title>
+  </head>
+  <body>
+    <p>
```
Comment (rabingaire) [inline index.html:6]: Why did you add p tag here?

Comment (rabingaire) [reply index.html:6]: Because this is a paragraph

**Review by pbbaidya — APPROVED** [review]
